app/Resources/assets/js/lsdoc/index.js

```diff
@@ -1,16 +1,33 @@
 $(document).on('ready', function(){
-    SaltGithub.getRepoList();
+    SaltGithub.init();
+    $('input[name="import"]').click(function(){
+        loadContent($('input[name="import"]:checked').val());
+    });
+    $('.import-framework').click(function(){
+        ImportFrameworks.fromAsn();
+    });
 });
 
 var SaltGithub = (function(){
 
+    function init(){
+        getRepoList();
+        $('#js-wizard-btn').click(function(){
+            loadModal();
+        });
+    }
+
     function getRepoList(){
         if( $('.js-github-list').length > 0 ){
-            $.get('/app_dev.php/user/github/files', function(data){
+            $.get('/app_dev.php/user/github/repos', function(data){
```

Comment:
We should only use /app_dev.php for development, as it will not work for normal users at some point after we have deployed to a production system.